src/components/Charts/LineChart/LineChart.tsx

```diff
@@ -23,13 +23,12 @@ export function LineChart({
   series,
   ...otherProps
 }: CustomLineChartProps) {
+  const theme = useMantineTheme();
+  const styles = getStyles(theme);
+
   return (
     <div style={styles.chartContainer}>
-      {title && (
-        <Typography variant="h3" style={styles.chartTitle}>
-          {title}
-        </Typography>
-      )}
+      {title && <div style={styles.chartTitle}>{title}</div>}
```

Comment:
How about give the title prop default empty string value and just render this div?